ledger/complete/wal/checkpoint_v6_reader.go

```diff
@@ -19,6 +20,8 @@ import (
 // ErrEOFNotReached for indicating end of file not reached error
 var ErrEOFNotReached = errors.New("expect to reach EOF, but actually didn't")
 
+var ReadTriesRootHash = readTriesRootHash
```

Comment:
We could. 

The private function is defined at the bottom of the file. I wanted to create this alias, so that all the public functions are listed at the beginning of the file, easy to browse all public functions.